clmm/modeling.py

```diff
@@ -484,7 +495,8 @@ def predict_convergence(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo, delt
 
     if z_src_model == 'single_plane':
         sigma_c = get_critical_surface_density(cosmo, z_cluster, z_source)
-        kappa = sigma / sigma_c
+        kappa = np.nan_to_num(sigma / sigma_c)
```

Comment:
Can we explicitly add the argument for nan (also for readability):
`kappa = np.nan_to_num(sigma / sigma_c, nan=0.0)`